### ISSUE TITLE
Comment/Question Obtaining Demo Dependencies 

### ISSUE THREAD

**Issue by achave11** [ga4gh/server#1464]
In [http://ga4gh-reference-implementation.readthedocs.io/en/latest/demo.html#demo](url) is it assumed that the user already has `wget` installed in order to run `(ga4gh-env) $ wget https://github.com/ga4gh/server/releases/download/data/ga4gh-example-data_4.5.tar` ??

**Comment by achave11** [comment]
For set up, also and if installing `wget` ensure latest version of X-code is running for MAC users. There is some incompatibility issues with OS-El Capitan and latter.

**Comment by david4096** [comment]
I don't think we need to use wget here, curl will be on both linux and mac systems and should work for this. We can replace with `curl -O`.
